common.js

```diff
@@ -0,0 +1,16 @@
+function Comma(Num) { //function to add commas to textboxes
```

Comment:
rename function to addComma or some such.

Reply:
done